doc/Type/Any.pod6

```diff
@@ -921,7 +921,7 @@ examples.
 
 Defined as:
 
-    method permutations(|c)
+    method permutations(Any: |c)
```

Comment:
Well, and I'm surprised here. You add 'Any:' which is really implicit, but not the tail? What gives?